vars/getJenkinsLabels.groovy

```diff
@@ -17,18 +17,22 @@ def call(String backend, String region=null, String datacenter=null) {
                           'gce-us-east1': 'gce-sct-builders-us-east1',
                           'gce-us-west1': 'gce-sct-builders-us-west1',
                           'gce': 'gce-sct-builders',
-                          'docker': 'sct-builders']
+                          'docker': 'sct-builders',
+                          'azure-eastus': 'aws-sct-builders-us-east-1-new']
```

Comment:
We have `azure-sct-builders` label and 1 builder is deployed already: https://jenkins.scylladb.com/computer/azure%2Dqa%2Dbuilder1/

Reply:
fixed in a separated PR, as we wanted to freeze this PR to get it merged:
https://github.com/scylladb/scylla-cluster-tests/pull/4506/commits/1f3ed9ea0f1efb9d9d94712103c10da655b5cc0f

Reply:
I think It can be done here, along with other small fixes.

Reply:
no objections